spec/factories/users.rb

```diff
@@ -1,7 +1,36 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  admin                  :boolean          default(FALSE)
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  type                   :string           default("Patient")
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
 FactoryBot.define do
-  factory :user , aliases: [:doctor, :patient] do
+  factory :patient, class: User do
     email { Faker::Internet.email }
-    password_digest {BCrypt::Password.create("123456")}
-    admin { [UserAdmin::FALSE] }
+    password { Faker::Internet.password }
+    password_confirmation { password }
```

Comment:
I've found it's easier to set passwords to the same thing for tests, something easy to remember like `Password1!`. That way you don't have to guess what they are in controller tests for example.